src/orbits/simple.jl

```diff
@@ -43,26 +45,47 @@ function relative_position(orbit::SimpleOrbit, t)
     return SA[x, y, z]
 end
 
+# TODO: if texp, ϵ += 0.5 * texp
+#function in_transit(orbit::SimpleOrbit, t)
+#    Δt = relative_time.(orbit, t)
+#    ϵ = 0.5 * orbit.duration
+#    return findall(x -> abs(x) < ϵ, Δt)
+#end
+#function in_transit(orbit::SimpleOrbit, t, r)
+#    Δt = relative_time.(orbit, t)
+#    ϵ = √((1.0 + r)^2.0 - orbit.b^2.0) / orbit.speed
+#    return findall(x -> abs(x) < ϵ, Δt)
+#end
+
 function flip(orbit::SimpleOrbit, ror)
     t0 = orbit.t0 + orbit.half_period
     b = orbit.b / ror
     speed = orbit.speed / ror
     ref_time = orbit.t0
-    return SimpleOrbit(orbit.period, t0, b, orbit.duration, speed, orbit.half_period, ref_time)
+    return SimpleOrbit(
+        orbit.period, t0, b, orbit.duration, speed, orbit.half_period, ref_time
+    )
```

Comment:
```suggestion
    return SimpleOrbit(orbit.period, t0, b, orbit.duration, speed, orbit.half_period, ref_time)
```
no need to change the formatting here